fixtures/v2/api/deleteRegistrantChange/success.http

```diff
@@ -0,0 +1,13 @@
+HTTP/2 201 
+server: nginx
+date: Tue, 22 Aug 2023 11:14:44 GMT
+content-type: application/json; charset=utf-8

```

Comment:
This doesn't seem correct, as the body is not JSON.

Reply:
That is the correct response we receive from the backend. The client should be able to handle these responses. However, I do agree that something is not right about the response, since the correct response would be 204.